docs/user-guide.md

```diff
@@ -295,22 +297,21 @@ Jakarta EE API configuration assistance is offered through Eclipse LSP4Jakarta,
+If this build does not complete successfully, the Liberty Tools function might not work correctly.
 
-E.g. one problem in particular can arise if your project uses a Gradle wrapper at a Gradle version less than v7.3 (and thus NOT supporting Java 17 per [matrix](https://docs.gradle.org/current/userguide/compatibility.html)), while using the Java 17+ workspace JRE required for running Liberty Tools.
+For example, if your project uses a Gradle wrapper at a Gradle version less than v7.3, it does not support Java 17 per [matrix](https://docs.gradle.org/current/userguide/compatibility.html), but the Java 17+ workspace JRE is required for running Liberty Tools.
 
-One approach for resolving this incompatibility is to configure the preference at:
-**Preferences -> Gradle -> Java home** to refer to a Java installation compatible with the level of Gradle in your `gradlew` wrapper.
+One approach to resolve this incompatibility is to configure the preference at:
+**Preferences -> Gradle -> Java home** to refer to a Java installation that is compatible with the level of Gradle in your `gradlew` wrapper.
 
-More generally the Gradle Eclipse preferences and project Java requirements must be compatible.
+More generally, the Gradle Eclipse preferences and project Java requirements must be compatible.
 
 
 ## Multi-module 
+Gradle multi-project builds are not supported at this time.
 
-There is no support for Gradle, multi-project builds at this time.
-
-We do offer a level of support for Maven, multi-module projects.  Rather than attempt to fully model the multi-project build, we make a heuristic "best guess" at the project structure, and allow the user to edit and customize the start command used.   We also may offer a choice of dashboard entries, one for the top-level, aggregate module and one for a specific server module.  The top-level module dashboard entry may not be usable in the case that Liberty dev mode is not able to guess the right starting module on its own, (see [here](https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#multiple-modules) for more info), in which case the server module entry should provide a useful starting point instead.
+Liberty Tools does offer a level of support for Maven, multi-module projects.  Rather than attempt to fully model the multi-project build, we make a heuristic "best guess" at the project structure, and allow the user to edit and customize the start command used.  Liberty Tools also might offer a choice of dashboard entries, one for the top-level, aggregate module and one for a specific server module.  The top-level module dashboard entry might not be usable in the case that Liberty dev mode is not able to guess the right starting module on its own, in which case the server module entry can provide a useful starting point instead. For more information, see the [Liberty Maven plugin multi-module documentation ](https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#multiple-modules).
```

Comment:
```suggestion
Liberty Tools offers a limited support for Maven multi-module projects. Rather than attempt to fully model the multi-project build, we make a heuristic "best guess" at the project structure, and allow the user to edit and customize the start command they use.  Liberty Tools also might offer a choice of dashboard entries, one for the top-level, aggregate module and one for a specific server module.  The top-level module dashboard entry might not be usable in the case that Liberty dev mode is not able to guess the right starting module on its own, in which case the server module entry can provide a useful starting point instead. For more information, see the [Liberty Maven plugin multi-module documentation ](https://github.com/OpenLiberty/ci.maven/blob/main/docs/dev.md#multiple-modules).
```